api-reference/10 UI Widgets/dxDiagram/3 Methods/reloadContent(itemKey_applyLayout).md

```diff
@@ -3,13 +3,20 @@ id: dxDiagram.reloadContent(itemKey, applyLayout)
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Reloads diagram data from a data source.
 
 ##### param(itemKey?): any
-<!-- Description goes here -->
+Specifies a data key of an item to reload.
+* If the parameter is specified the widget reloads the corresponding item and every added and deleted items.
```

Comment:
Can you clarify what you mean by "every added and deleted items"?

Reply:
If the parameter is specified, the widget does not search differences in existing items, but it updates the specified item,  inserted items, and deleted items

Reply:
```suggestion
* If the parameter is specified, the widget reloads the corresponding item, inserted items and deleted items.
```